src/mpyl/steps/postdeploy/cypress_test.py

```diff
@@ -0,0 +1,61 @@
+""" Step that runs relevant cypress tests in the post deploy stage """
+
+import os
+
+from logging import Logger
+from typing import cast
+
+from python_on_whales import docker, Container
+
+from .. import Step, Meta
+from ..models import ArtifactType, Input, Output
+from ...project import Stage
+from ...utilities.docker import execute_with_stream, stream_encoded_logging
+
+
+class CypressTest(Step):
+    def __init__(self, logger: Logger) -> None:
+        super().__init__(logger, Meta(
+            name='Cypress Test',
+            description='Step to run cypress tests',
+            version='0.0.1',
+            stage=Stage.POST_DEPLOY
+        ), produced_artifact=ArtifactType.NONE, required_artifact=ArtifactType.NONE)
+
+    def execute(self, step_input: Input) -> Output:
+        self._logger.info(f"Running cypress tests for project {step_input.project.name}")
+
+        cypress_config = step_input.run_properties.config['cypress']
+        if not cypress_config:
+            raise ValueError("Cypress config is required when cypress test step is used")
+
+        volume_path = os.path.join('.', cypress_config['volumePath'])
+        if os.getcwd().endswith('tests'):
+            volume_path = './test_resources/cypress'
```

Comment:
Make sure the step also works when running the (disabled) test and building the test projects, by setting the volume path to the one in the test resources.

Reply:
I don't think we should put that in mainline code. Can't you set the `volume_path` via the config class?

Reply:
so I thought I could just use the value from `mpyl_config` but then it doesn't work in the pipeline build (for mpyl pr's) since that uses the config from the separate repo which should contain the path of the monorepo. 

any suggestions?

Reply:
Ah, yeah, I was gonna say, this should come from the dependencies 👍 2e33087926e2716c308cfb1db517566282fd3953